packages/devextreme/js/ui/chat.d.ts

```diff
@@ -0,0 +1,170 @@
+import Widget, { WidgetOptions } from './widget/ui.widget';
+import {
+    Cancelable,
+    EventInfo,
+    NativeEventInfo,
+    InitializedEventInfo,
+    ChangedOptionInfo,
+} from '../events/index';
+
+/**
+ * @docid _ui_chat_DisposingEvent
+ * @public
+ * @type object
+ * @inherits EventInfo
+ */
+export type DisposingEvent = EventInfo<dxChat>;
+
+/**
+ * @docid _ui_chat_InitializedEvent
+ * @public
+ * @type object
+ * @inherits InitializedEventInfo
+ */
+export type InitializedEvent = InitializedEventInfo<dxChat>;
+
+/**
+ * @docid _ui_chat_OptionChangedEvent
+ * @public
+ * @type object
+ * @inherits EventInfo,ChangedOptionInfo
+ */
+export type OptionChangedEvent = EventInfo<dxChat> & ChangedOptionInfo;
+
+/**
+ * @docid _ui_chat_MessageSendEvent
+ * @public
+ * @type object
+ * @inherits Cancelable,NativeEventInfo,Message
+ */
+export type MessageSendEvent = Cancelable & NativeEventInfo<dxChat, KeyboardEvent | PointerEvent | MouseEvent | TouchEvent> & Message;
+
+/**
+ * @docid
+ * @namespace DevExpress.ui.dxChat
+ * @public
+ */
+export type User = {
+    /**
+     * @docid
+     * @default undefined
+     * @public
+     */
+    id?: number;
+    /**
+     * @docid
+     * @default ''
+     * @public
+     */
+    name?: string;
+    /**
+     * @docid
+     * @default ''
+     * @public
+     */
+    avatarUrl?: string;
+};
+
+/**
+ * @docid
+ * @namespace DevExpress.ui.dxChat
+ * @public
+ */
+export type Message = {
+    /**
+     * @docid
+     * @default ''
+     * @public
+     */
+    timestamp?: string;
```

Comment:
For history: It's requires to think about name of this option and its accepted values